scss/_utilities.scss

```diff
@@ -473,6 +473,7 @@ $utilities: map-merge(
         lighter: $font-weight-lighter,
         normal: $font-weight-normal,
         bold: $font-weight-bold,
+        semibold: $font-weight-semibold,
```

Comment:
Check in Bootstrap if `fw-semibold` wasn't defined for examples. Otherwise remove it in Boosted.